<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemEventPageWidget</class>
 <widget class="QWidget" name="SystemEventPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemEventPageWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QProgressBar" name="parsingProgressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="eventsTreeView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Event IDs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eventIdLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="applyPushButton">
           <property name="toolTip">
            <string>Apply filter</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="plugin.qrc">
             <normaloff>:/plugin/images/Resources/apply.png</normaloff>:/plugin/images/Resources/apply.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="errorCheckBox">
           <property name="text">
            <string>Error</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="debugCheckBox">
           <property name="text">
            <string>Debug</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="fatalCheckBox">
           <property name="text">
            <string>Fatal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="infoCheckBox">
           <property name="text">
            <string>Info</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="warnCheckBox">
           <property name="text">
            <string>Warning</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectAllPushButton">
           <property name="toolTip">
            <string>Select all</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="plugin.qrc">
             <normaloff>:/plugin/images/Resources/selectAll.png</normaloff>:/plugin/images/Resources/selectAll.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectNonePushButton">
           <property name="toolTip">
            <string>Unselect all</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="plugin.qrc">
             <normaloff>:/plugin/images/Resources/unselectAll.png</normaloff>:/plugin/images/Resources/unselectAll.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QListView" name="providersListView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextBrowser" name="descriptionTextBrowser"/>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Creation time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="sourceLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="creationTimeLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="plugin.qrc"/>
 </resources>
 <connections/>
</ui>
